packages/app/src/components/uiBlocks/TagSearchResult.vue

```diff
@@ -0,0 +1,105 @@
+<template>
+  <div class="px-20 py-8">
+    <div
+      v-for="tag in searchResult"
+      :key="tag.nfcData"
+      class="flex pb-6"
+      :class="{ 'opacity-25 ': selectedTag !== tag && selectedTag !== null }"
+      @click="$emit('tag-selected', tag)"
+    >
+      <TagEntry :img="tag.imageUrl" class="w-20 h-20 flex-shrink-0" />
+      <div class="pl-3 flex flex-col text-white overflow-hidden flex-grow">
+        <span class="text-lg font-bold whitespace-nowrap overflow-hidden overflow-ellipsis">
+          {{ tag.name }}
+        </span>
+        <span
+          v-if="tag.trackArtists"
+          class="text-base whitespace-nowrap overflow-hidden overflow-ellipsis"
+        >
+          {{ tag.trackArtists }} - {{ tag.trackTitle }}
+        </span>
+      </div>
+    </div>
+  </div>
+</template>
+
+<script lang="ts">
+import { Paginated } from '@feathersjs/feathers';
+import { NFCTag } from '@leek/commons';
+import { defineComponent, onMounted, PropType, ref, watchEffect } from 'vue';
+
+import feathers from '../../lib/feathers';
+import TagEntry from './TagEntry.vue';
+
+export default defineComponent({
+  name: 'TagSearchResult',
+
+  components: {
+    TagEntry,
+  },
+
+  props: {
+    selectedTag: {
+      type: Object as PropType<NFCTag | null>,
+      default: null,
+    },
+    searchInput: {
+      type: String,
+      default: '',
+    },
+  },
+
+  emits: {
+    // eslint-disable-next-line @typescript-eslint/no-unused-vars
+    'tag-selected': (_payload: NFCTag): boolean => true,
+  },
+
+  setup(props) {
+    const allTags = ref<NFCTag[]>([]);
+    const searchResult = ref<NFCTag[]>([]);
```

Comment:
Later we can (should change this)
In the PR #224 Implement UI for editing a tag I added a service to get the 

Then:

 const selectedTrack = ref<SpotifyTrack>();